### PR TITLE
Fetch support

### PR DESCRIPTION
**Pull request by stevelacey**
Adds support for fetch in addition to xhr (and jquery) #421

Most notably (for me anyway) this gets debugbar working in Apollo-based SPA's and GraphiQL.

### CODE REVIEW

**Comment by stevelacey** [comment]
Note that due to the way we have to swap out `window.fetch` rather than the `open` method on the xmlhttprequest prototype, most users will probably need to stick `async defer` on their script(s) to track initial requests on pageload; something for docs maybe... I imagine it'd apply to a way large percent of use cases than with `xhr` given the common use-case for fetch being in clients like `Apollo` that initialize on pageload and touch `window.fetch` then rather than on the first request.

File: src/DebugBar/Resources/debugbar.js [hunk 1]
```diff
@@ -1037,31 +1037,49 @@ if (typeof(PhpDebugBar) == 'undefined') {
...
-         * @param {XMLHttpRequest} xhr
+         * @param {Response|XMLHttpRequest} response
          * @return {Bool}
          */
-        handle: function(xhr) {
-             // Check if the debugbar header is available
-            if (xhr.getAllResponseHeaders().indexOf(this.headerName) === -1){
+        handle: function(response) {
+            // Check if the debugbar header is available
+            if (this.isFetch() && !response.headers.has(this.headerName + '-id')) {
```
Comment (barryvdh) [inline src/DebugBar/Resources/debugbar.js]: Does this need a response? `this.isFetch(response)`

**Comment by barryvdh** [comment]
Did you test this with bot xhr and fetch? Is this ready?

**Comment by stevelacey** [comment]
@barryvdh can you tag a release 🙏

**Comment by barryvdh** [comment]
I tagged it, but doesn't yet show on packagist. @maximebf can you check your Packagist integration settings?

**Comment by stevelacey** [comment]
@barryvdh might be GitHub, they're going through Partially Degraded Service right now

**Comment by stevelacey** [comment]
Maybe not, @maximebf help?

**Comment by stevelacey** [comment]
@maximebf pint

**Comment by stevelacey** [comment]
@barryvdh should note that the version constraints between the last tagged version on package and master are quite different, the minimum PHP version is bumped from 5.3 to 5.6 for one thing.

Looks like there hasn't been a tagged release in almost 2 years 😱

**Comment by stevelacey** [comment]
@maximebf ping

**Pull request merged**